fhir-codegen/gradle-plugin/src/main/kotlin/com/google/fhir/codegen/FhirCodegenTask.kt

```diff
@@ -172,51 +164,11 @@ abstract class FhirCodegenTask : DefaultTask() {
     // Generates a wrapper for enum types
     EnumerationFileSpecGenerator.generate(packageName).writeTo(outputDir)
 
-    generateSharedEnums(valueSetMap, packageName, outputDir)
-
     // Generate custom serializers
     val serializersPackageName = "$packageName.serializers"
     DoubleSerializerFileSpecGenerator.generate(serializersPackageName).writeTo(outputDir)
     LocalTimeSerializerFileSpecGenerator.generate(serializersPackageName).writeTo(outputDir)
 
     FhirJsonTransformerFileSpecGenerator.generate(packageName).writeTo(outputDir)
   }
-
-  /**
-   * Generate shared enums. These enum classes are created from [StructureDefinition.snapshot]
-   * Elements that have common binding extensions.
-   */
-  private fun generateSharedEnums(
```

Comment:
might make sense just to create a new file Enum....FhirSpecGenerator or something.